moto/kms/models.py

```diff
@@ -263,6 +275,7 @@ def __init__(self, region_name: str, account_id: Optional[str] = None):
         super().__init__(region_name=region_name, account_id=account_id)  # type: ignore
         self.keys: Dict[str, Key] = {}
         self.key_to_aliases: Dict[str, Set[str]] = defaultdict(set)
+        self.multi_region_configuration: dict = {"MultiRegionConfiguration": {}}
```

Comment:
This doesn't seem to be used anywhere? The value is set in the `replicate_key`-method, but it's not read anywhere afterwards.